src/content/configuration/index.mdx

```diff
@@ -286,7 +286,7 @@ module.exports = {
       // properites that are read from description file
       // to alias requests in this package
       enforceExtension: false,
-      // if true request must not include an extensions
+      // if true request must not include an extension
```

Comment:
```suggestion
      // if true, request must not include an extension
```
like this @EugeneHlushko ?

Reply:
yes, but on the other hand i am not sure it is important, this is a comment in JavaScript example snippet